kasa/iotprotocol.py

```diff
@@ -51,33 +39,27 @@ async def _query(self, request: str, retry_count: int = 3) -> Dict:
         for retry in range(retry_count + 1):
             try:
                 return await self._execute_query(request, retry)
-            except httpx.CloseError as sdex:
-                await self.close()
+            except httpx.ConnectError as sdex:
                 if retry >= retry_count:
-                    _LOGGER.debug("Giving up on %s after %s retries", self.host, retry)
+                    _LOGGER.debug("Giving up on %s after %s retries", self._host, retry)
                     raise SmartDeviceException(
-                        f"Unable to connect to the device: {self.host}: {sdex}"
+                        f"Unable to connect to the device: {self._host}: {sdex}"
```

Comment:
Would you mind separating rename-only (or similar) changes into another PR that we could review and merge immediately? That'll make it easier to deal with this, as it's grown to be quite wild..

Reply:
So the rename only above and the `_XorTransport` and `__init__` signatures test below in one separate PR is ok?

Reply:
Yeah, so any minor changes (especially renames or unrelated refactorings, just for example) that are not directly related to the connectionparameters should be in their own. Basically anything that is a logically standalone change should be done in their own PRs, that helps to keep the PRs small and faster to merge. No need to go overboard with splitting up when it would inflict too much pain, but it's a good practice to keep it small & simple :-)

Reply:
Actually how would you feel about me adding the renames and `__init__` signature stuff to https://github.com/python-kasa/python-kasa/pull/580?  I'll be cleaning up the transports to remove the `needs_login` and `needs_handshake` so I think it will be simpler to have that all in the same place.

Reply:
Sounds good to me :+1:

Reply:
Done in https://github.com/python-kasa/python-kasa/pull/580